monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -270,9 +270,9 @@ set +e
 get_ingress_ports
 
 # get URLs for Grafana, Prometheus and AlertManager
-gf_url=$(get_service_url $MON_NS v4m-grafana  "false")
-# pr_url=$(get_url $MON_NS v4m-prometheus  "false")
-# am_url=$(get_url $MON_NS v4m-alertmanager  "false")
+gf_url=$(get_service_url $MON_NS v4m-grafana  "$TLS_ENABLE")
+# pr_url=$(get_url $MON_NS v4m-prometheus  "$TLS_ENABLE")
```

Comment:
Just out of curiosity, should these be commented out, or is there a plan to uncomment in the future?

Reply:
We used to make these other web applications accessible by default.  However, we stopped doing that a while back since they lack any built-in authentication capability.  I suspect we should probably just remove the commented out lines but didn't feel comfortable doing that without further discussion/thought.